src/js/components/devices/terminal.js

```diff
@@ -9,27 +9,32 @@ import { SearchAddon } from 'xterm-addon-search';
 import msgpack5 from 'msgpack5';
 
 import { setSnackbar } from '../../actions/appActions';
+import { deviceConnect } from '../../actions/deviceActions';
 
 // see https://github.com/mendersoftware/go-lib-micro/tree/master/ws
 //     for the description of proto_header and the consts
 // *Note*: this needs to be aligned with mender-connect and deviceconnect.
-const MessageProtocolShell = 1;
+export const MessageProtocol = {
+  Shell: 1
+};
 
-const MessageTypeNew = 'new';
-const MessageTypePing = 'ping';
-const MessageTypePong = 'pong';
-const MessageTypeResize = 'resize';
-const MessageTypeShell = 'shell';
-const MessageTypeStop = 'stop';
+export const MessageTypes = {
+  New: 'new',
+  Ping: 'ping',
+  Pong: 'pong',
+  Resize: 'resize',
+  Shell: 'shell',
+  Stop: 'stop'
+};
```

Comment:
lovely! I quess the 'delay' comes in the following commits.

Reply:
in the _other_ terminal PR there is some additional work to bring those into the device constants... I'll rebase there and will adjust accordingly - including the error and stop codes 👌